app/config/wiring.scala

```diff
@@ -54,7 +55,7 @@ object WSHttp extends WSHttp with Hooks
 
 class GmpSessionCache @Inject()(environment: Environment,
```

Comment:
Only keep the GmpSessionCache and rename this file to match class name...Delete everything else